packages/calcite-components/src/components/action-bar/action-bar.tsx

```diff
@@ -292,6 +295,13 @@ export class ActionBar extends LitElement {
 
   private handleDefaultSlotChange(): void {
     this.updateGroups();
+    if (this.layout === "horizontal-fill") {
```

Comment:
@josercarcamo I think this needs a little bit more work to handle when the layout changes. If the layout changes on the component, we should update the fullWidth property. Maybe something like:

```
a.fullWidth = this.layout === "horizontal-fill";
```

There should also be a way to update these when there are "changes" to layout. If a user switches from horizontal-fill to something else the actions should be updated.